src/molgenis/capice_resources/process_vep/__main__.py

```diff
@@ -110,16 +124,29 @@ def run_module(self, arguments):
         train_features = self._read_train_features(arguments['features'])
         cgd = self._read_cgd_data(arguments['genes'])
         build38 = arguments['assembly']
+        previous_iteration = arguments['train_test_previous_iteration']
+        if previous_iteration is not None:
+            previous_iteration_dataset = self._read_pandas_tsv(
+                previous_iteration,
+                ['CHROM', 'POS', 'REF', 'ALT', 'SYMBOL']
```

Comment:
records might reference the same gene through different gene symbols (e.g. an alias). is the information available to match on the combination of gene source + gene identifier?